### PR TITLE
OT_ME: merge called functions into caller

### PR DESCRIPTION
**Pull request by toxeus**

### CODE REVIEW

**Comment by mhamrle** [comment]
:+1: except one comment

File: src/client/commands/CmdNewOffer.cpp [hunk 1]
```diff
@@ -344,8 +345,9 @@ int32_t CmdNewOffer::cleanMarketOfferList(
         otOut << "Canceling market offer with transaction number: " << id
               << ".\n";
 
+        OT_ME ot_me;
         string response =
-            MadeEasy::kill_market_offer(server, mynym, myacct, id);
+            ot_me.kill_market_offer(server, mynym, myacct, std::stoi(id));
```
Comment (mhamrle) [inline src/client/commands/CmdNewOffer.cpp]: stoi return int not int64

**Comment by mhamrle** [comment]
I will merge it after rebasing and fixing `stoi`

**Comment by toxeus** [comment]
@mhamrle fixed and rebased

**Pull request merged**